tests/test_smoke_tests.py

```diff
@@ -52,13 +52,14 @@ def test_that_exploitable_crash_report_not_displayed_for_not_logged_in_users(sel
         Assert.false(csp.header.is_exploitable_crash_report_present)
 
     @pytest.mark.credentials
```

Comment:
Good catch, thanks! Must have slipped back in during one of my edits .. poof! Gone :-)